portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,126 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import java.util.List; 
+import java.util.ArrayList; 
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  int numQuoteToDisplay = 10;
+
+  /**
+    * This method retrives all quotes that are stored in Datastore
+    * and puts them into a list of quotes. This list is then written
+    * as the response from the server. 
+    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Quote").addSort("timestamp", SortDirection.DESCENDING);
+    PreparedQuery quotes = datastore.prepare(query);
+
+    //Add each quote to the quote list whose content will be written as 
```

Comment:
Let's have a space after //. Here and below.

ex:

// Add each quote ...

Reply:
Done.